source/reference/verification.txt

```diff
@@ -16,117 +16,104 @@ Verify Data Transfer
    :depth: 2
    :class: singlecol
 
-When :ref:`mongosync <c2c-mongosync>` has fully committed, verify the 
-successful transfer of your data before you switch your application to using 
-the destination cluster. You can verify your data transfer using document 
-counts, hash comparison, document comparison, or the Migration Verifier.
+Before you switch your application load from the source cluster
+to the destination cluster, you should always verify that the
+sync was successful.
 
-Use Cases
----------
-
-You should verify your data after every sync. This is important in
-cases where you plan to move your application load from the source to the
-destination cluster.
-
-Verification methods:
-
-- :ref:`c2c-verify-with-doc-counts`
-
-- :ref:`c2c-verify-with-hash-comp`
-
-- :ref:`c2c-verify-with-doc-comp`
-
-- :ref:`c2c-verify-with-verifier`
-
-The specific method you use to verify your data depends on your application
-workload and the complexity of the data.
+Starting in 1.9, ``mongosync`` provides an embedded verifier
+that can perform a series of verification checks on the source
+and destination cluster to ensure a successful sync.
```

Comment:
"ensure a successful" -> "confirm that the sync was successful"

Reply:
Fixed